src/editor/extended_python_generator.ts

```diff
@@ -31,6 +31,8 @@ import * as commonStorage from '../storage/common_storage';
 export class ExtendedPythonGenerator extends PythonGenerator {
   private currentModule: commonStorage.Module | null = null;
   private mapWorkspaceIdToExportedBlocks: { [key: string]: Block[] } = Object.create(null);
+  protected methods_: {[key: string]: string} = Object.create(null);
```

Comment:
If this generator is used to generate code for multiple blocks workspaces (for example, the workspace in the UI as well as headless workspaces that we use for creating new modules), will this methods_ object get messed up?

Reply:
At this point, I'm ok with this being a problem that we create an issue for it. Then we can think about what the best solution is. I'll create an issue and put in some notes for discussion.